stable/datacube/templates/wms-deployment.yaml

```diff
@@ -59,12 +77,12 @@ spec:
         - name: DB_USERNAME
           valueFrom:
             secretKeyRef:
-              name: {{ template "datacube.secretName" . }}
+              name: {{ .Values.database.existingSecret }}
               key: postgres-username
         - name: DB_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: {{ template "datacube.secretName" . }}
+              name: {{ .Values.database.existingSecret }}
```

Comment:
This should be removed, the template datacube.secretName already points to existingSecret (see _helpers.tpl) it also includes a default function that you have now removed.

Reply:
Reverted the changes back to `template "datacube.secretName" .`